.github/workflows/ci.yml

```
@@ -32,16 +32,20 @@ jobs:
       - name: Linting
         run: uv run ruff check -- src/
 
-      - name: Run tests locally
-        run: uv run pytest tests
+      # - name: Run tests locally
+      #   run: uv run pytest tests
 
       # Note: This spins up containers running the default services.
       - name: Spin up Docker Compose stack in background
         run: docker compose up --detach
 
       # Note: This spins up the "test" container.
+      - name: Run ingest script
+        run: docker compose --profile=tools run ingest  
```

Just passing through (I saw the "PR" number increase on the GitHub UI). This comment has become inconsistent with the code below it.

```suggestion
      # Note: This spins up the "ingest" container.
      - name: Run ingest script
        run: docker compose --profile=tools run ingest  
```